### PR TITLE
updating caption

### PR DESCRIPTION
**Pull request by tstanekovic94**
# Updated Caption Pull Request

This is a pull request for the updated captions

#1

### CODE REVIEW

**Comment by abhishekraghavtjx** [comment]
all good

File: _slides/08-tstanekovic94.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is my updated caption
```
Comment (KevinK88TJX) [inline _slides/08-tstanekovic94.md:6]: Comment added

**Review by KevinK88TJX — APPROVED** [review]

File: _slides/08-tstanekovic94.md (hunk 1 above)
Comment (abhishekraghavtjx) [inline _slides/08-tstanekovic94.md:6]: this is fantastic

**Review by abhishekraghavtjx — APPROVED** [review]
well done Tay!

**Pull request merged**